### ISSUE TITLE
Consider using Resume directive on DelimiterFraming to drop oversized lines

### ISSUE THREAD

**Issue by drewhk** [akka/akka#18515]
I.e. if supervision with Resume is used, just start dropping incoming bytes from the oversized line until new delimiter is found.

**Comment by ktoso** [comment]
Sounds like a good idea :+1:

**Comment by rkuhn** [comment]
I wouldn’t tie this to supervision, though.

**Comment by fatihcataltepe** [comment]
This would be a really useful feature.

**Comment by ktoso** [comment]
PRs very welcome @fatihcataltepe :)

**Comment by He-Pin** [comment]
@patriknw Is this still valid?

**Comment by johanandren** [comment]
Yeah, I think this could still be nice, as Roland said, it should probably not really be supervision though, but rather a flag when constructing a delimiter based frame decoder.
